client/browser-tests/route-load-tests-config.js

```diff
@@ -190,4 +190,14 @@ module.exports.route_load_tests_config = [
     route: "resource-explorer",
     test_on: ["eng", "basic-eng"],
   },
+  {
+    name: "Data Lab landing page",
```

Comment:
It's not called the data lab, it's the InfoLab!!!!!11

Reply:
`<jk>` Too late, you approved it so I'm merging it as is `</jk>`